Lib/pdb.py

```diff
@@ -1073,6 +1093,24 @@ def _select_frame(self, number):
         self.print_stack_entry(self.stack[self.curindex])
         self.lineno = None
 
+    def do_exceptions(self, arg):
+        if not arg:
```

Comment:
what is the type of arg?

Reply:
For archival purpose, see other comment, `arg` is a str, that represent the number of the current exception. Marking as resolved.